### PR TITLE
Update starkgate-automated_actions_with_bridging.adoc

### PR DESCRIPTION
**Pull request by natan-granit**
### Description of the Changes

Please add a detailed description of the change, whether it's an enhancement or a bugfix.
If the PR is related to an open issue please link to it.

### PR Preview URL

After you push a commit to this PR, a preview is built and a URL to the root of the preview appears in the comment feed.

Paste here the specific URL(s) of the content that this PR addresses.

### Check List

- [x] Changes have been done against master branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1045)

### CODE REVIEW

File: components/Starknet/modules/tools/pages/starkgate-automated_actions_with_bridging.adoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [id="con_automated_actions_with_bridging"]
-= Automated actions with bridging
-
+= Performing a Smart Deposit
+// please move to be under procedures
```
Comment (stoobie) [inline components/Starknet/modules/tools/pages/starkgate-automated_actions_with_bridging.adoc:3]:
```suggestion

```

**Review by stoobie — DISMISSED** [review]
LGTM

**Review by stoobie — DISMISSED** [review]
LGTM

File: components/Starknet/modules/tools/pages/starkgate-automated_actions_with_bridging.adoc [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
+The `depositWithMessage` function enables a Smart Deposit. `depositWithMessage` is similar to the `deposit` function, with an additional 256-bit message, which can contain instructions for executing additional actions.
 
-Upon completion, the `depositWithMessage` function triggers a call to a callback function, named `on_receive`, on the receiving L2 contract. The `on_receive` function receives the deposit message as input.
+Upon completion, the `depositWithMessage` function triggers a call to a callback function, named `on_receive`, on the L2 contract that receives the deposit. The `on_receive` function receives the deposit message as input.
 
 `on_receive` must return `true` for the deposit to succeed. If `on_receive` returns `false`, or if the recipient contract does not include the `on_receive` function, the `depositWithMessage` function's L1 handler fails. The user can recover their funds using the `depositWithMessageCancelRequest` function.
 
+.To enable this feature in your application:
```
Comment (stoobie) [inline components/Starknet/modules/tools/pages/starkgate-automated_actions_with_bridging.adoc:12]:
```suggestion
.To enable Smart Deposits in your application:
```

File: components/Starknet/modules/tools/pages/starkgate-automated_actions_with_bridging.adoc [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
+
+. Implement the `on_receive` function in the L2 that should receive deposits.
```
Comment (stoobie) [inline components/Starknet/modules/tools/pages/starkgate-automated_actions_with_bridging.adoc:14]:
```suggestion
. Implement the `on_receive` function in the L2 contract that should receive deposits.
```

**Review by natan-granit — COMMENTED** [review]
LGTM, I appreciate the details in incorporating the Smart Deposit term.

**Pull request merged**